UI.tsv

```diff
@@ -325,7 +325,7 @@ UI_20150317_000324	{@st56}Dead{/}
 UI_20150317_000325	Spell Power
 UI_20150317_000326	Defense
 UI_20150317_000327	Magic Resistance
-UI_20150317_000328	Critical Rate
+UI_20150317_000328	Critical Attack Chance
```

Comment:
Disagreed. 'Critical Rate' at any rate is not a very clear term. A rate that is critical, in a literal sense. You're mixing several terms that are very game specific with adjectives that are self-defined, for example, AoE is clearly a game term, whereas 'Pierce' or 'Blunt' is a term defined by dictionaries and has an explanatory meaning. 'Critical Rate' is a new terminology that is based on games, meaning imbued by players/users of the game community and is not a defined term for ordinary people. 'Critical Attack chance' does not do full justice but does alleviate the explanation of what this number/percentage may be. Since I have not seen if whether this is indicated as a % value or a number value, I have left it as 'chance', but this can be changed to either term 'rate' or 'value'.
If there is a help documentation in-game, that would be a good place to add explanation for each stat names. I'd rather have in-game section explaining everything about games rather than one having to search on wiki's.
I'm sure IMC will add an in-game help section/glossary where people can open and read the detailed explanations. Establishing and referring to wiki pages created by users does not necessarily guarantee that the information contained within are correct.